userguide/dailyuse/libertine.rst

```diff
@@ -194,6 +194,17 @@ Add lines like the following ones to your ``~/.bash_aliases``::
 
     alias git='libertine-launch -i CONTAINER-IDENTIFIER git'
     alias screenfetch='libertine-launch -i CONTAINER-IDENTIFIER screenfetch'
+    
+Installing packages manually
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you want to install a package from a file, you will first have to make it available to the root user, for example by copying it to the ``/root`` directory::
+
+    cp ~/Downloads/somepackage.deb ~/.cache/libertine-container/CONTAINER-IDENTIFIER/rootfs/
```

Comment:
I don't think this puts it into /root, I think it puts it into / no?!

Reply:
@maciek134 ping

Reply:
pong ;)

Reply:
actually, isn't ~/Downloads mounted from the host into the container anyway ...

Reply:
Uh, somehow this got into spam and I totally forgot. I'll try to set up libertine on my device tomorrow and verify all that.

Reply:
Awesome

Reply:
Fixed the `/root` part. I don't think `~/Downloads` is mounted by default, tried on a fresh install with a firefox deb and `# find / -name firefox\*` didn't return anything.

Reply:
> I don't think `~/Downloads` is mounted

huh that's strange. here we are claiming that it is: https://github.com/ubports/docs.ubports.com/blob/master/userguide/dailyuse/libertine.rst#files Could you do me a favor and double check? It would be so much nicer to just download in UT as normal, then enter the container and do `dpkg -i ~/Downloads/somepackage.deb` If you don't have `Downloads` could you check whether you have `Documents` or the others Pictures Videos Music

Reply:
well, whatever. this should work. If it can be done nicer maybe that will come later